test/_data/flutter_style_todos/a.dart

```diff
@@ -0,0 +1,22 @@
+// Copyright (c) 2018, the Dart project authors. Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+// test w/ `pub run test -N flutter_style_todos`
+
+// BAD
+//TODO
+// TODO
+// TODO: bla
+// TODO(user) remove
+// TODO:(user): Button
+// TODO(user) : We
+/// nothing here
```

Comment:
DBC: Oh man, I wish this weren't in the "bad" category. This means I can't use it, because I use js style doc comments.

```
/// public docs
class ....
```

Reply:
I'm not sure to understand. There's no lint on L14, only on line 15

Reply:
Ah, should this be moved into "GOOD" then?